api-testing/multicloud-api/src/main/java/com/soda/utils/okhttputils/OkHttpRequests.java

```diff
@@ -0,0 +1,110 @@
+/*
+  Copyright 2020 The SODA Authors.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+ */
+package com.soda.utils.okhttputils;
+
+import com.soda.utils.Logger;
+import okhttp3.*;
+
+import java.io.IOException;
+
+public abstract class OkHttpRequests {
```

Comment:
Why do we need this class & methods but itself the methods from the library can be used directly?
Are we handling the request specifically for this project?

I see all the method body very similiar.

Reply:
This is repeated questions. Already discuss with @kumarashit

Reply:
https://github.com/Click2cloud-Wadhai can you please put the reasons over here

Reply:
@himanshuvar  This is abstract class. I have use for only showing functionality  so it will reduce the code size, Otherwise if i have written all request call in HttpHandler the lines of code  is increase and it is very difficult to find out a particular code.